docs/src/faq.md

```diff
@@ -0,0 +1,402 @@
+```@meta
+CurrentModule = Enzyme
+DocTestSetup = quote
+    using Enzyme
+end
+```
+
+# Frequently asked questions
+
+## Implementing pullbacks
+
```

Comment:
Maybe also say or forwardmode?

Reply:
wdym?

Reply:
I think it would be confusing because we speak about combined reverse mode just below. Pushforwards are readily available with Enzyme.Forward

Reply:
Ah I forgot the title of this was "implementing pullbacks", yeah I just meant that duplicated returns are valid in split mode and also forward mode (but not in combined reverse mode atm)